### ISSUE TITLE
Chroma injects unexpected line breaks in function names when parsing C++

### ISSUE THREAD

**Issue by manvis** [alecthomas/chroma#124]
I've used Chroma both from within Hugo and command line. Try this simple code:
```
int main() {
    return 0;
}
```
When the language is set to C, Chroma creates correct HTML. When the language is set to C++, I start seeing unexpected line breaks.

![bugreport](https://user-images.githubusercontent.com/4654412/36850778-4b1ed5a8-1d70-11e8-8e0e-5c1bef646ec5.png)

**Comment by alecthomas** [comment]
I think that might be caused by d7ee3c10b04394e072a13fae63698f62d9cb3006. Can you try the commit before that one and see if that fixes it? If so, I'll revert it for now.

**Comment by manvis** [comment]
I've tried to reset to before the commit you indicated. Unfortunately, the problem seems to be elsewhere. The problem occurs even if I output html without line numbers or if I use the terminal formatter. Moreover, it seems that only C++ function names are affected.

It is possible I'm mixing something up because I'm not too familiar with golang (and especially building applications). However, considering that ` display: block; ` got removed from the ` LineNumbersTable ` style in the exported css, I assume I did things correctly.
